arithmetization/src/main/java/net/consensys/linea/zktracer/module/shakiradata/ShakiraData.java

```diff
@@ -90,4 +90,22 @@ public void commit(List<MappedByteBuffer> buffers) {
       operation.trace(trace, ++stamp);
     }
   }
+
+  @Override
+  public String toString() {
```

Comment:
What's the purpose ?

Reply:
just to make it readible if when we need to log it. We can delete though as it's not logged atm.